azure-pipelines.yml

```diff
@@ -1,19 +1,23 @@
-# Starter pipeline
-# Start with a minimal pipeline that you can customize to build and deploy your code.
-# Add steps that build, run tests, deploy, and more:
-# https://aka.ms/yaml
-
 trigger:
-- main
+  - main
 
-pool:
-  vmImage: ubuntu-latest
+pr:
+  autoCancel: 'true'
 
-steps:
-- script: echo Hello, world!
-  displayName: 'Run a one-line script'
+resources:
+  repositories:
+    - repository: templates
+      type: git
+      name: nhsuk.tools.packages.pipeline.templates
 
-- script: |
-    echo Add other tasks to build, test, and deploy your project.
-    echo See https://aka.ms/yaml
-  displayName: 'Run a multi-line script'
+extends:
+  template: main.yml@templates
+  parameters:
+    AgentImageName: 'ubuntu-latest'
+    PackageFeedProjectName: 'nhsuk/nhsuk.tools'
+    PackageFeedName: 'nhsuk.tools'
```

Comment:
I don't believe we should need these two parameters as I think they're only used for the Azure package feeds?

Reply:
of course yes, i forgot to remove them. will update. 
also i just saw  that the PublishStorybook task runs in the PR . so i need to only allow if it isMain

Reply:
resolved